recipes/fwdpp/meta.yaml

```diff
@@ -20,14 +20,14 @@ requirements:
     - boost >=1.60.0
 
 build:
-  number: 0
+  number: 1
 package:
   name: fwdpp
-  version: '0.5.3'
+  version: '0.5.4'
 test:
   commands:
-    - fwdppConfig --version | grep 0.5.3 > /dev/null
+    - fwdppConfig --version | grep 0.5.4 > /dev/null
 source:
-  fn:  0.5.3.tar.gz 
-  url: http://github.com/molpopgen/fwdpp/archive/0.5.3.tar.gz
-  md5: 09a00eb8fa1d54c953b5cb4e52a85a1c
+  fn:  0.5.4rc1.tar.gz 
+  url: http://github.com/molpopgen/fwdpp/archive/0.5.4rc1.tar.gz
```

Comment:
I think I misunderstood the build logs.  I was expecting to see GCC output, output from 'make check', etc.  But it seems to indicate that the build passed.  Is this correct?  (It'd really be nice to see more verbose output, like for a "normal" Travis build.)

If this is working, I'll release 0.5.4 and update this PR with 0.5.4 as the final version.